### PR TITLE
Added more memory tier with proper resource access

### PR DESCRIPTION
**Pull request by bamachrn**
This is replacement of PR https://github.com/codeready-toolchain/host-operator/pull/944

### CODE REVIEW

File: pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go [hunk 1]
```diff
@@ -599,11 +599,27 @@ func assertNamespaceTemplate(t *testing.T, decoder runtime.Decoder, actual templ
...
 		templatePath = fmt.Sprintf("%s/ns_%s.yaml", tier, typeName)
 	}
+	t.Logf("checking template '%s' (based on another tier: %t)", templatePath, basedOnOtherTier(expectedTiers, tier))
 	content, err := assets.Asset(templatePath)
 	require.NoError(t, err)
 	expected := templatev1.Template{}
 	_, _, err = decoder.Decode(content, nil, &expected)
 	require.NoError(t, err)
+	// then override the templates' parameters (if applicable)
+	if basedOnOtherTier(expectedTiers, tier) {
```
Comment (alexeykazakov) [inline pkg/templates/nstemplatetiers/nstemplatetier_generator_whitebox_test.go:609]: I'm not sure why this is needed. Could you please comment on this?

**Review by alexeykazakov — COMMENTED** [review]
It looks like there are problems with unit tests for this PR.

**Review by MatousJobanek — COMMENTED** [review]
could you please answer why you changed the templates in `test/templates/` folder?
There is no point in changing that right? It's not a problem that they don't align with the actual ones - they are just testing "dummy" templates.

**Comment by MatousJobanek** [comment]
This PR hasn't been updated for a long time and there are a tons of conflicts. Closing it... 
If you still want to change the values then please open a new PR.

**Pull request closed without merging**